projects/mac_ai/testing/test.py

```
@@ -27,8 +27,15 @@ def prepare_ci():
     """
     Prepares the cluster and the namespace for running the tests
     """
-
-    return prepare_mac_ai.prepare()
+    try:
+        return prepare_mac_ai.prepare()
+    except Exception:
+        if not config.project.get_config("remote_host.run_locally"):
+            # retrieve all the files that have been saved remotely
+            exc = run.run_and_catch(exc, run.run_toolbox, "remote", "retrieve",
```

Flake8 (7.2.0)

[error] local variable 'exc' is assigned to but never used

(F841)